### DISCUSSION TITLE
Preferred development язык

### DISCUSSION THREAD

**Discussion by S-trace** [xtrime-ru/antizapret-vpn-docker#10] [General]
Which language should be preferred here in discussions, commit messages and pull requests - english или русский?

**Comment by xtrime-ru** [comment]
Я думаю, что русский предпочтителен, так как данный сервис актуален только для РФ. Readme на английском я сделал по привычке...

**Reply by S-trace** [reply]
Ну яхез, [this guy](https://github.com/Nefelum/antizapret-vpn-docker) added many non-russian domains - it seems this project is useful outside of РФ too...

**Reply by S-trace** [reply]
Я просто собираюсь написать гайд по настройке современных версий OpenWRT в сочетании с PiHole и думаю на каком языке писать)

**Reply by xtrime-ru** [reply]
оригинальная дока и форум тоже на русском: https://ntc.party/c/antizapret-prostovpn-org/az-self-hosted/38

Я все же склоняюсь к русскому.
